src/lib.ts

```diff
@@ -111,25 +113,14 @@ export function safeInterrupt(): void {
   }
 }
 
-export function persistEmbezzlerLog(): { total: number; sources: Array<string> } {
+export function resetDailyPreference(trackingPreference: string): boolean {
```

Comment:
based on the way  I see this getting used it might be worth doing
```suggestion
export function resetDailyPreference(trackingPreference: string, ...dependentPreferences: string[]): boolean {
```
It also might not!

Reply:
I didn't do this because I thought that it would e cleaner to have it return a Boolean so the user could set their own defaults instead of forcing them to use empty string